pkg/sources/git/git.go

```diff
@@ -319,6 +322,48 @@ type cloneParams struct {
 	clonePath string
 }
 
+var ExecutableName = "trufflehog"
+
+func getScannerPIDs(ctx context.Context) ([]int, error)  {
+	pids:= []int{}
+
+	procs, err := ps.Processes()
+	if err != nil {
+		ctx.Logger().Error(err, "Error getting job PIDs")
+		return nil, err
+	}
+
+	for _, proc := range procs {
+		if strings.Contains(proc.Executable(), ExecutableName) {
+			pids = append(pids, proc.Pid())
+		}
+	}
+
+	return pids, nil
+}
+
+func cleanTempDir(ctx context.Context, pid int) error {
```

Comment:
this function is in kind of a weird place: some of its errors it returns, but others it logs. you could get consistency by either logging all the errors and returning nothing or using [errors.Join](https://pkg.go.dev/errors#Join) to combine deletion errors into a single error and returning that. @ahrav what's your opinion on this?